File: WitchcraftApp/src/test/java/com/tarbadev/witchcraft/converter/TbspToCupConverterTest.java
package com.tarbadev.witchcraft.converter;

import org.junit.Before;
import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class TbspToCupConverterTest {
  private TbspToCupConverter subject;

  @Before
  public void setUp() {
    subject = new TbspToCupConverter();
  }

  @Test
  public void convert() {
    Double tbsp = 1.0;
    Double expectedCup = 0.0625;

    assertThat(subject.convert(tbsp)).isEqualTo(expectedCup);

    tbsp = 4.0;
    expectedCup = 0.25;

    assertThat(subject.convert(tbsp)).isEqualTo(expectedCup);
  }
}

File: WitchcraftApp/src/test/java/com/tarbadev/witchcraft/converter/TbspToCupConverterTest.kt
package com.tarbadev.witchcraft.converter

import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.assertEquals

class TbspToCupConverterTest {
    private lateinit var tbspToCupConverter: TbspToCupConverter

    @BeforeEach
    fun setUp() {
        tbspToCupConverter = TbspToCupConverter()
    }

    @Test
    fun convert() {
        var tbsp = 1.0
        var expectedCup = 0.0625

        assertEquals(expectedCup, tbspToCupConverter.convert(tbsp))

        tbsp = 4.0
        expectedCup = 0.25

        assertEquals(expectedCup, tbspToCupConverter.convert(tbsp))
    }
}